pipeline_dp/aggregate_params.py

```diff
@@ -113,45 +113,83 @@ def metrics_str(self) -> str:
             return f"custom combiners={[c.metrics_names() for c in self.custom_combiners]}"
         return f"metrics={[m.value for m in self.metrics]}"
 
+    @property
+    def bounds_per_value_are_set(self) -> bool:
```

Comment:
Optional: Maybe `bounds_per_contribution_are_set`? In one case, we are bounding each contribution, in the other we are bounding total contribution to a partition.